package/src/components/Message/MessageSimple/MessageSimple.tsx

```diff
@@ -234,7 +249,26 @@ const MessageSimpleWithContext = <
             {message.pinned ? <MessagePinnedHeader /> : null}
           </View>
 
-          <View style={[styles.contentWrapper, contentWrapper]}>
+          <Swipeable
+            containerStyle={[styles.contentWrapper, contentWrapper]}
+            friction={2}
+            leftThreshold={enableSwipeToReply ? 100 : 0}
+            onSwipeableWillOpen={() => {
+              if (!swipeableRef.current) return;
+              clearQuotedMessageState();
+              setQuotedMessageState(message);
+              triggerHaptic('impactLight');
+              swipeableRef.current.close();
+            }}
+            ref={swipeableRef}
+            renderLeftActions={() => {
+              if (alignment === 'left' && enableSwipeToReply) {
```

Comment:
Do we only support swipe-to-reply if the alignment of the received messages is to the left ?

Reply:
Yes, iOS SDK does the same. It should work only for un deleted message for left aligned messages.

Reply:
So, if an integrator has an inverted alignment of messages they can't get this functionality ?

Reply:
Why would they have it? Do you mean the message sent on the left side and the received messages on the right side?

Reply:
I mean having [this](https://getstream.io/chat/docs/sdk/react-native/ui-components/message-simple/#alignment) set so that you achieve what you said, yeah - this would prevent the integrators from having this feature altogether.

The silver lining is it should be easy to add here, right ?

Reply:
They can disable the feature for such a case because its unlikely they would want it as they would have a UI similar to Slack.

Reply:
Hmm I don't think we should support only half of our feature set - I would add that functionality too (especially considering it's easy to do)